README.md

```diff
@@ -36,13 +36,18 @@ transXpress requires:
 
 1. Install [Miniconda3](https://conda.io/en/latest/miniconda.html)
 
-2. Setup conda environment (optional):
+2. Unset R_LIBS_SITE
```

Comment:
Please add a brief explanation why this is needed

Reply:
is this explanation ok? 

> To ensure correct versions of R packages will be used unset R_LIBS_SITE

Reply:
Yes